docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -65,18 +76,16 @@ The following example shows how to create a gallery-like flexbox layout using th
     toolBar.JustifyContent = FlexContainer.Justification.JustifySpaceBetween;
     ```
 
-5.  Use the toolbar's `Flex` property to make the toolbar and the main content share the height of the main container, so that the toolbar occupies 10 percent of the vertical space and the content occupies the remainder:
+6.  Use `Flex` property of the toolbar to specify occupied area in main container by it. The main container's height is shared between toolbar and content (added in step 7). Since main container displays items vertically and toolbar `Flex` property is set to 0.1, toolbar will occupy 10% of verical and 100% of horizontal available space. Remainded area is intended for content:
 
```

Comment:
@nawab-samsung I changed description of this subpoint a little. Please share your opinion wheter usage of `Flex` property is clearly explained here.

Reply:
@pkalota sure will check